query/stdlib/influxdata/influxdb/to.go

```diff
@@ -499,7 +499,7 @@ type Stats struct {
 	NTags    int
 }
 
-func (s Stats) Update(o Stats) {
+func (s Stats) Update(o Stats) Stats {
```

Comment:
Bug: This method does a lot of updates to the fields of `s`, but since it wasn't being passed as a reference none of the updates were having an effect. I decided to keep the pass-by-value here and return the new data because at the callsite `s` is a member of a `map`, so getting a reference to it is a pain.